### PR TITLE
I have added the footer

### LINKED ISSUE #3: create footer

### PR DESCRIPTION
**Pull request by Akram-baloch**

### CODE REVIEW

File: book-store/layouts/footer.php [hunk 1]
```diff
@@ -0,0 +1,99 @@
+  <!-- FOOTER START -->
```
Comment (MuslimJaan) [inline book-store/layouts/footer.php:1]: please format this cod

File: book-store/layouts/footer.php [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    </div>
+    <div class="container-fluid bg-black">
+
+      <div class="container p-5">
+
+        <div class="row">
+          <div class="col-sm-6">
+            <div class="d-flex w-50 justify-content-between">
+
+              <img src="images/payment_01.jpg" alt="">
```
Comment (MuslimJaan) [inline book-store/layouts/footer.php]: Image not found in alt

File: book-store/layouts/footer.php [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+              <img src="images/payment_02.jpg" alt="">
```
Comment (MuslimJaan) [inline book-store/layouts/footer.php]: Image not found in alt

File: book-store/layouts/footer.php [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+              <img src="images/payment_03.jpg" alt="">
```
Comment (MuslimJaan) [inline book-store/layouts/footer.php]: Image not found in alt

File: book-store/layouts/footer.php [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+              <img src="images/payment_04.jpg" alt="">
```
Comment (MuslimJaan) [inline book-store/layouts/footer.php]: Image not found in alt

File: book-store/layouts/footer.php [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+            </div>
+
+          </div>
+        </div>
+      </div>
+
+    </div>
+    <div class="container-fluid bg-black">
+
```
Comment (MuslimJaan) [inline book-store/layouts/footer.php]: please remove spase

File: book-store/layouts/footer.php [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+    <div class="container-fluid bg-black pt-5 mt-5">
+      <div class="container p-5 bg-black">
+        <div class="row">
+          <div class="col-sm-3">
+            <h6 class="text-uppercase text-light fw-bold mb-4">connect</h6>
+            <P>
+              <i class="text-decoration-none text-secondary" href="">2005 Stokes Isle Apt. 896,</i>
+            </P>
+            <P>
+              <i class="text-decoration-none text-secondary" href="">Venaville 10010, USA</i>
```
Comment (MuslimJaan) [inline book-store/layouts/footer.php:12]: please add # in href

File: book-store/layouts/footer.php [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+            </P>
+            <P>
+              <i class="text-decoration-none text-secondary" href="">info@yourdomain.com</i>
```
Comment (MuslimJaan) [inline book-store/layouts/footer.php:15]: please add # in href

File: book-store/layouts/footer.php [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+            </P>
+            <p><i class="text-decoration-none text-secondary" href="">(+68) 120034509</i></p>
```
Comment (MuslimJaan) [inline book-store/layouts/footer.php:17]: please add # in href

File: book-store/layouts/footer.php [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+              <i class="bi-facebook h5"></i>
+              <i class="bi-twitter h5"></i>
+              <i class="bi-instagram h5"></i>
+              <i class="bi-skype h5"></i>
+            </div>
+          </div>
+          <div class="col-sm-3">
+            <h6 class="text-uppercase text-light mb-4">extra link</h6>
+            <p>
+              <i class="text-decoration-none text-secondary" href="">Affiliate Program</i>
```
Comment (MuslimJaan) [inline book-store/layouts/footer.php:28]: please add # in href

File: book-store/layouts/footer.php [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+            </p>
+            <p>
+              <i class="text-decoration-none text-secondary" href="">Business Services
```
Comment (MuslimJaan) [inline book-store/layouts/footer.php:31]: please add # in href

File: book-store/layouts/footer.php [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+              </i>
+            </p>
+            <p>
+              <i class="text-decoration-none text-secondary" href="">Education Services
```
Comment (MuslimJaan) [inline book-store/layouts/footer.php:35]: please add # in href

File: book-store/layouts/footer.php [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+              </i>
+            </p>
+            <p>
+              <i class="text-decoration-none text-secondary" href=""> Gift Cards
```
Comment (MuslimJaan) [inline book-store/layouts/footer.php:39]: please add # in href

File: book-store/layouts/footer.php [hunk 14]
```diff
@@ -0,0 +1,99 @@
...
+              </i>
+            </p>
+          </div>
+          <div class="col-sm-3">
+            <h6 class="text-uppercase text-light mb-4">your account</h6>
+            <p><i class="text-secondary" href="">Shop</i></p>
```
Comment (MuslimJaan) [inline book-store/layouts/footer.php]: please add # in href

File: book-store/layouts/footer.php [hunk 15]
```diff
@@ -0,0 +1,99 @@
...
+            <p><i class="text-secondary" href="">My Orders</i></p>
```
Comment (MuslimJaan) [inline book-store/layouts/footer.php]: please add # in href

File: book-store/layouts/footer.php [hunk 16]
```diff
@@ -0,0 +1,99 @@
...
+            <p><i class="text-secondary" href="">Gift Collections</i></p>
```
Comment (MuslimJaan) [inline book-store/layouts/footer.php]: please add # in href

File: book-store/layouts/footer.php [hunk 17]
```diff
@@ -0,0 +1,99 @@
...
+            <p><i class="text-secondary" href="">Purchase Cards</i></p>
```
Comment (MuslimJaan) [inline book-store/layouts/footer.php]: please add # in href

File: book-store/layouts/footer.php [hunk 18]
```diff
@@ -0,0 +1,99 @@
...
+            <p><i class="text-secondary" href="">FAQs</i></p>
```
Comment (MuslimJaan) [inline book-store/layouts/footer.php]: please add # in href

**Review by MuslimJaan — CHANGES_REQUESTED** [review]

File: book-store/layouts/footer.php [hunk 19]
```diff
@@ -0,0 +1,99 @@
...
+                <i class="text-danger bi-star-fill h5"></i>
+                <i class="text-danger bi-star-fill h5"></i>
+                <i class="text-danger bi-star-fill h5"></i>
+                <i class="text-secondary bi-star h5"></i>
+                <p>$49.00 - $60.00</p>
+
+              </div>
+
+            </div>
+
```
Comment (Farazbhai) [inline book-store/layouts/footer.php]: Please remove this line

File: book-store/layouts/footer.php [hunk 20]
```diff
@@ -0,0 +1,99 @@
...
+                <p>Henry & The Good Dog</p>
+                <i class="text-danger bi-star-fill h5"></i>
+                <i class="text-danger bi-star-fill h5"></i>
+                <i class="text-danger bi-star-fill h5"></i>
+                <i class="text-danger bi-star-fill h5"></i>
+                <i class="text-secondary bi-star h5"></i>
+                <p>$49.00 - $60.00</p>
+
+              </div>
+
```
Comment (Farazbhai) [inline book-store/layouts/footer.php]: Remove empty line

File: book-store/layouts/footer.php [hunk 21]
```diff
@@ -0,0 +1,99 @@
...
+              <img class="w-25" src="../Assets/images/f-img.jpg" alt="">
+              <div class="">
+                <p>Henry & The Good Dog</p>
+                <i class="text-danger bi-star-fill h5"></i>
+                <i class="text-danger bi-star-fill h5"></i>
+                <i class="text-danger bi-star-fill h5"></i>
+                <i class="text-danger bi-star-fill h5"></i>
+                <i class="text-secondary bi-star h5"></i>
+                <p>$49.00 - $60.00</p>
+
```
Comment (Farazbhai) [inline book-store/layouts/footer.php]: Remove empty line

File: book-store/layouts/footer.php [hunk 22]
```diff
@@ -0,0 +1,99 @@
...
+              </div>
+
+            </div>
+
+          </div>
+        </div>
+      </div>
+
```
Comment (Farazbhai) [inline book-store/layouts/footer.php:68]: Remove this empty line

File: book-store/layouts/footer.php (hunk 6 above)
Comment (Farazbhai) [inline book-store/layouts/footer.php]: Remove empty line

File: book-store/layouts/footer.php [hunk 23]
```diff
@@ -0,0 +1,99 @@
...
+    </div>
+    <div class="container-fluid bg-black">
+
+      <div class="container p-5">
+
```
Comment (Farazbhai) [inline book-store/layouts/footer.php]: Remove empty line

File: book-store/layouts/footer.php [hunk 24]
```diff
@@ -0,0 +1,99 @@
...
+        <div class="row">
+          <div class="col-sm-6">
+            <div class="d-flex w-50 justify-content-between">
+
```
Comment (Farazbhai) [inline book-store/layouts/footer.php]: Remove empty line

File: book-store/layouts/footer.php [hunk 25]
```diff
@@ -0,0 +1,99 @@
...
+            </div>
+          </div>
+          <div class="col-sm-6 text-end">
+            <i class="text-secondary">© 2018 Auteur. All Right Reserved.</i>
+          </div>
+        </div>
+      </div>
+    </div>
+  </footer>
+
```
Comment (Farazbhai) [inline book-store/layouts/footer.php]: Remove empty line

File: book-store/layouts/footer.php [hunk 26]
```diff
@@ -0,0 +1,99 @@
...
+  <!-- FOOTER END -->
+
```
Comment (Farazbhai) [inline book-store/layouts/footer.php]: Remove empty line

File: book-store/layouts/footer.php [hunk 27]
```diff
@@ -0,0 +1,99 @@
...
+  </body>
+
```
Comment (Farazbhai) [inline book-store/layouts/footer.php]: Remove empty line

**Review by Farazbhai — APPROVED** [review]

**Review by MuslimJaan — APPROVED** [review]

**Pull request merged**